python/pyspark/sql/functions.py

```diff
@@ -6594,20 +6594,28 @@ def substring_index(str: "ColumnOrName", delim: str, count: int) -> Column:
 
 
 @try_remote_functions
-def levenshtein(left: "ColumnOrName", right: "ColumnOrName") -> Column:
+def levenshtein(
+    left: "ColumnOrName", right: "ColumnOrName", threshold: Optional[int] = None
+) -> Column:
     """Computes the Levenshtein distance of the two given strings.
 
     .. versionadded:: 1.5.0
 
     .. versionchanged:: 3.4.0
         Supports Spark Connect.
 
+    .. versionchanged:: 3.5.0
+        Supports Spark Connect.
```

Comment:
I prefer another `versionadded ` after parameter `threshold`, you can refer to https://github.com/apache/spark/blob/ab4693d979b4879ca07268e3719c20a5088e87ec/python/pyspark/sql/pandas/map_ops.py#L55-L67

Reply:
This is done.